src/Components/Components/ref/Microsoft.AspNetCore.Components.csproj

```diff
@@ -7,13 +7,11 @@
     <Compile Include="Microsoft.AspNetCore.Components.netstandard2.0.cs" />
     <Reference Include="Microsoft.Extensions.Logging.Abstractions"  />
     <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions"  />
-    <Reference Include="Microsoft.Extensions.HashCodeCombiner.Sources"  />
```

Comment:
Need to update the *.Manual.cs files. As-is I expect compilation errors due to missing `internal` classes for some (if not many) uses of the ref/ assemblies

Reply:
The compilations errors have been resolved.